scim-sdk-server/src/main/java/de/captaingoldfish/scim/sdk/server/patch/workarounds/msazure/MsAzurePatchComplexValueRebuilder.java

```diff
@@ -157,4 +150,30 @@ public PatchRequestOperation fixPatchRequestOperaton(ResourceType resourceType,
     operation.setValues(fixedValues);
     return operation;
   }
+
+  private boolean shouldBeFixed(String jsonDocument)
+  {
+    if (StringUtils.isBlank(jsonDocument))
+    {
```

Comment:
just one thing. 
I have checked out the code and verified everything. Please document the method so that it will be easier to understand what it does in one year or more. What it does is not absolutely obvious when we look at it for the first time.

Reply:
Agreed. I refactored the code a little and added some docs. Please check.